atest/testdata/standard_libraries/builtin/repeat_keyword.robot

```diff
@@ -31,6 +32,12 @@ Invalid Times 2
     [Documentation]    FAIL STARTS: 'notaninteger' cannot be converted to an integer: ValueError:
     Repeat Keyword    Not an integer    No Operation
 
+Repeat Keyword With Time String
+    Repeat Keyword    00:00:00.3    Keyword Executing For 0.1 second    00:00:00.3
+    Repeat Keyword    0.3 seconds    Keyword Executing For 0.1 second    0.3 seconds
+    Repeat Keyword    0.3 secs    Keyword Executing For 0.1 second    0.3 secs
+    Repeat Keyword    0.3s    Keyword Executing For 0.1 second    0.1s 200x
```

Comment:
@pekkaklarck Thanks for your opinion.
OK, I will change the test code just verifying only more than one executing to simplify and shorten the test.
I wanted to test 'x' as millisecond not a count if it is in time string, so that I wrote 0.1second and 200 millisecond (totally 0.3 second)